products/storage/ansible.yaml

```diff
@@ -45,7 +45,7 @@ overrides: !ruby/object:Provider::ResourceOverrides
 examples: !ruby/object:Api::Resource::HashArray
 files: !ruby/object:Provider::Config::Files
   compile:
-<%= lines(indent(compile('provider/ansible/common~compile.yaml'), 4)) -%>
+<%= lines(indent(compile('provider/ansible/product~compile.yaml'), 4)) -%>
```

Comment:
It looks like all the ansible products are using a common~compile step, so shouldn't you be able to just remove these lines and let the common logic kick in rather then renaming them to product compile? I also don't see `provider/ansible/product~compile.yaml` in this PR

Reply:
It's in there! It's just a rename so it's easy to miss on GitHub.

The Ansible compiled files are all compiled with product-specific attributes afaik - so `product~compile` is a better name now given how we've changed TF.